ajax/libs/heapbox/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "heapbox",
+  "version": "0.9.4",
+  "filename": "jquery.heapbox-0.9.4.min.js",
```

Comment:
You can see https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-common-conventions
> Filenames should not include a version number and be lowercase.

Or auto-update can't work well.
Please discuss with the author about the change of naming scheme, or we might have to manually add the file afterwards.